data/map.txt

```diff
@@ -14526,6 +14593,8 @@ system "Kor Ak'Mari"
 	trade Metal 217
 	trade Plastic 524
 	fleet "Korath Raid" 2800
+	fleet "Korath Miners" 5000
+	fleet "Korath Large Raid" 1500
```

Comment:
```suggestion
	fleet "Korath Large Raid" 2500
```
Kor Ak'Mari is *extremely* busy now for being an empty system, it should probably be toned down a bit.